LibRangeCheck-3.0/LibRangeCheck-3.0.lua

```diff
@@ -977,103 +1011,157 @@ function lib:init(forced)
 end
 
 --- Return an iterator for checkers usable on friendly units as (**range**, **checker**) pairs.
-function lib:GetFriendCheckers()
-  return rcIterator(self.friendRC)
+-- @param inCombat if true, only checkers that can be used in combat ar returned
+function lib:GetFriendCheckers(inCombat)
```

Comment:
The addition of an parameter is technically not fully backwards compatible, as the caller could have used a bogus extra parameter. But I find that an acceptable thing to potentially break.

Also, here I choose to not change what GetFriendCheker etc returns. The other choice would have been to make inCombat the default.

So addons might need to adjust their call depending on what they want. The returned checker might not work in combat, but will not cause an lua error .

Reply:
I think that's fine.  @kodewdle does ElvUI need to adjust for this?

Reply:
ElvUI don't use this call.